numpy/distutils/fcompiler/gnu.py

```diff
@@ -315,6 +315,12 @@ def version_match(self, version_string):
     module_dir_switch = '-J'
     module_include_switch = '-I'
 
+    if sys.platform[:3] == 'aix':
```

Comment:
If this stays in this file (referring to my question on BLAS), this is okay - although by design the better place is probably to add the method `get_flags_linker_so` et al. - but given the number of sets of flags to be edited, that's not necessarily more understandable this way.

Reply:
Okay. I want to make the additions in one place rather than in various places.